yt_idefix/_io/commons.py

```diff
@@ -47,6 +47,54 @@ def padded(self) -> Coordinates:
         return Coordinates(arrs[0], arrs[1], arrs[2], self.array_shape)
 
 
+def mapFromCart(
```

Comment:
avoid `camelCase` in Python, and prefer `snake_case_without_shortened_names`
I assume a better name here would be `get_native_coordinates_from_cartesian` ?

Reply:
Done!